.github/workflows/accessibility.yml

```
@@ -0,0 +1,46 @@
+
+jobs:
+  accessibility:
+    name: Run Accessibility Checks
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node
+        if: matrix.language == 'javascript-typescript'
```

actionlint (1.7.4)

property "language" is not defined in object type {}

(expression)